### PR TITLE
docs(comparison): clarify some algolia points

### PR DESCRIPTION
**Pull request by Haroenv**
- virtual replicas allow for sorting without the surcharge of duplicating the records: https://www.algolia.com/doc/guides/managing-results/refine-results/sorting/in-depth/relevant-sort/#step-one---create-a-virtual-replica-index
- the default max document size is 100kb, but used to be smaller

### CODE REVIEW

**Comment by guimachiavelli** [comment]
Thanks for this, @Haroenv, it is much appreciated!

@ferdi05, looks fine to me on a first glance but perhaps you want to take a look as well?

**Review by dichotommy — APPROVED** [review]
LGTM, unless @ferdi05 or @gmourier have something to say about it.

**Comment by ferdi05** [comment]
Leaving the review to @gmourier

**Comment by dichotommy** [comment]
We have approval across the board so I'm merging this. Thanks @Haroenv ! 🔥 🚀

**Comment by dichotommy** [comment]
bors merge

**Pull request merged**
